test/NuGet.Clients.Tests/NuGet.VisualStudio.Implementation.Test/NuGet.VisualStudio.Implementation.Test.csproj

```diff
@@ -53,7 +53,8 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" NoWarn="NU1605" />
-  </ItemGroup>
+    <PackageReference Include="Microsoft.ServiceHub.Framework" NoWarn="NU1605"/>
```

Comment:
Not related with this PR. We have the pattern to NoWarn on downgrades.  Why we do not lift the versions and remove the NoWarn? All the warnings could be a blocker if we ever consider to adopt CPVM and these downgrades are from transitive packages.

Reply:
The packages are not there.
This is happening because we have to depend on preview packages because the publicly available ones don't have the features we need.

Reply:
So to be more explicit...these packages are developed in private feeds, every once in a while they are pushed to public feeds, and even more rarely they end up on nuget.org. 

There's many teams and many packages floating around so there's no combination of the packages being available, and the features we need available in public feeds that don't cause an NU1605. 
I try to upgrade them occasionally, but it's very difficult to get all teams to push all their packages.